<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WAAPI工具集</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblWaapiPort">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>WAAPI网络端口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spbWaapiPort">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>1000</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>8080</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnWaapiConnect">
        <property name="text">
         <string>连接到Wwise工程</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnGetSelectedObjects">
        <property name="text">
         <string>获取Wwise工程选中对象</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMultiEditor">
        <property name="text">
         <string>多项编辑器</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBatchRename">
        <property name="text">
         <string>批量重命名</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tblActiveObjects">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <column>
       <property name="text">
        <string>名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>路径</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbxKeepSelf">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>保留原始</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbxRecursiveFind">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>递归查找</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFindParent">
        <property name="text">
         <string>查找上级对象</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFindChildren">
        <property name="text">
         <string>查找下级对象</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnUpdateObjects">
        <property name="text">
         <string>更新列表内容</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemoveSelection">
        <property name="text">
         <string>删除列表选中</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClearObjects">
        <property name="text">
         <string>清空列表</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="cbbDescendantType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>全部类型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Action</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ActorMixer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AudioFileSource</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AuxBus</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BlendContainer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bus</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Event</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Folder</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GameParameter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MusicPlaylistContainer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MusicSegment</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MusicSwitchContainer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MusicTrack</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RandomSequenceContainer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sound</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SoundBank</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>State</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Switch</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SwitchContainer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WorkUnit</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="2">
       <layout class="QGridLayout" name="gridLayout_9"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="iptSelectionFilter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string extracomment="筛选文本"/>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <layout class="QGridLayout" name="gridLayout_10"/>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QRadioButton" name="rbnFilterByName">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>81</width>
           <height>19</height>
          </rect>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>按名称</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="rbnFilterByPath">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>0</y>
           <width>81</width>
           <height>19</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>按路径</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>20</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QRadioButton" name="rbnFilterByInclude">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>81</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>筛选</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="rbnFilterByExclude">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>0</y>
           <width>91</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>筛除</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="cbxCaseSensitive">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>区分大小写</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="cbxMatchWholeWord">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>全字匹配</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="cbxUseRegularExpression">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>使用正则表达式</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnuAudioSource">
    <property name="title">
     <string>AudioSource</string>
    </property>
    <addaction name="actApplyEditsToOriginal"/>
    <addaction name="actResetSourceEdits"/>
    <addaction name="actTrimTailSilence"/>
    <addaction name="separator"/>
    <addaction name="actRenameOriginalToWwise"/>
    <addaction name="actDeleteUnusedAKDFiles"/>
    <addaction name="actLocalizeLanguages"/>
   </widget>
   <widget class="QMenu" name="mnuLogicContainers">
    <property name="title">
     <string>Container</string>
    </property>
    <widget class="QMenu" name="mnuSwitchContainer">
     <property name="title">
      <string>Switch Container</string>
     </property>
     <addaction name="actAssignSwitchMappings"/>
     <addaction name="actRemoveAllSwitchAssignments"/>
     <addaction name="actSetGenericPath"/>
    </widget>
    <addaction name="actBreakContainer"/>
    <addaction name="actReplaceParent"/>
    <addaction name="actApplyFaderEditsDownstream"/>
    <addaction name="mnuSwitchContainer"/>
    <addaction name="actSplitByNetRole"/>
   </widget>
   <widget class="QMenu" name="mnuSoundBank">
    <property name="title">
     <string>SoundBank</string>
    </property>
    <widget class="QMenu" name="mnuSetInclusionTo">
     <property name="title">
      <string>调整包含内容</string>
     </property>
     <addaction name="actIncludeMediaOnly"/>
     <addaction name="actIncludeEventsAndStructures"/>
     <addaction name="actIncludeAll"/>
    </widget>
    <addaction name="actAddToSelectedBank"/>
    <addaction name="actCreateOrAddToBank"/>
    <addaction name="actBankAssignmentMatrix"/>
    <addaction name="separator"/>
    <addaction name="actClearInclusions"/>
    <addaction name="actCalculateBankSize"/>
    <addaction name="mnuSetInclusionTo"/>
   </widget>
   <widget class="QMenu" name="mnuCommon">
    <property name="title">
     <string>通用</string>
    </property>
    <widget class="QMenu" name="mnuRename">
     <property name="title">
      <string>改变命名规则</string>
     </property>
     <addaction name="actChangeToTitleCase"/>
     <addaction name="actChangeToLowerCase"/>
     <addaction name="actChangeToUpperCase"/>
    </widget>
    <widget class="QMenu" name="mnuConvertToType">
     <property name="title">
      <string>改变类型为</string>
     </property>
     <addaction name="actConvertToActorMixer"/>
     <addaction name="actConvertToBlendContainer"/>
     <addaction name="actConvertToRandomSequenceContainer"/>
     <addaction name="actConvertToSwitchContainer"/>
     <addaction name="actConvertToVirtualFolder"/>
     <addaction name="actConvertToWorkUnit"/>
    </widget>
    <widget class="QMenu" name="mnuCreateParent">
     <property name="title">
      <string>创建父级</string>
     </property>
     <addaction name="actCreateActorMixer"/>
     <addaction name="actCreateBlendContainer"/>
     <addaction name="actCreateRandomSequenceContainer"/>
     <addaction name="actCreateSwitchContainer"/>
     <addaction name="actCreateVirtualFolder"/>
     <addaction name="actCreateWorkUnit"/>
    </widget>
    <addaction name="mnuConvertToType"/>
    <addaction name="mnuRename"/>
    <addaction name="mnuCreateParent"/>
    <addaction name="separator"/>
    <addaction name="actMoveListToSelection"/>
    <addaction name="actCopySelectionToList"/>
    <addaction name="separator"/>
    <addaction name="actBatchReplaceTool"/>
   </widget>
   <widget class="QMenu" name="menuEvent">
    <property name="title">
     <string>Event</string>
    </property>
    <addaction name="actCreatePlayEvent"/>
   </widget>
   <widget class="QMenu" name="mnuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actImportFromFile"/>
    <addaction name="actExportToFile"/>
   </widget>
   <widget class="QMenu" name="mnuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
    <widget class="QMenu" name="mnuFilterByInclusion">
     <property name="title">
      <string>筛选</string>
     </property>
     <addaction name="actFilterIncluded"/>
     <addaction name="actFilterExcluded"/>
    </widget>
    <widget class="QMenu" name="mnuSetInclusion">
     <property name="title">
      <string>设为</string>
     </property>
     <addaction name="actSetIncluded"/>
     <addaction name="actSetExcluded"/>
    </widget>
    <addaction name="actUndo"/>
    <addaction name="actRedo"/>
    <addaction name="separator"/>
    <addaction name="mnuFilterByInclusion"/>
    <addaction name="mnuSetInclusion"/>
    <addaction name="actDeleteObjects"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>其他</string>
    </property>
    <addaction name="actTempTool"/>
    <addaction name="actUnityResourceManager"/>
   </widget>
   <addaction name="mnuFile"/>
   <addaction name="mnuEdit"/>
   <addaction name="mnuCommon"/>
   <addaction name="mnuAudioSource"/>
   <addaction name="mnuLogicContainers"/>
   <addaction name="menuEvent"/>
   <addaction name="mnuSoundBank"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEnable">
   <property name="text">
    <string>Enable</string>
   </property>
  </action>
  <action name="actionDisable">
   <property name="text">
    <string>Disable</string>
   </property>
  </action>
  <action name="actionEnable_2">
   <property name="text">
    <string>Enable</string>
   </property>
  </action>
  <action name="actionDisable_2">
   <property name="text">
    <string>Disable</string>
   </property>
  </action>
  <action name="actApplyEditsToOriginal">
   <property name="text">
    <string>应用剪辑至源文件</string>
   </property>
  </action>
  <action name="actResetSourceEdits">
   <property name="text">
    <string>重置剪辑</string>
   </property>
  </action>
  <action name="actAssignSwitchMappings">
   <property name="text">
    <string>自动分配</string>
   </property>
  </action>
  <action name="actionEnable_3">
   <property name="text">
    <string>Enable</string>
   </property>
  </action>
  <action name="actionDisable_3">
   <property name="text">
    <string>Disable</string>
   </property>
  </action>
  <action name="actDeleteObjects">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actCalculateBankSize">
   <property name="text">
    <string>计算媒体资源大小</string>
   </property>
  </action>
  <action name="actCreateOrAddToBank">
   <property name="text">
    <string>为每个对象创建或加入Bank</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionBreak">
   <property name="text">
    <string>Break</string>
   </property>
  </action>
  <action name="actChangeToTitleCase">
   <property name="text">
    <string>首字母大写</string>
   </property>
  </action>
  <action name="actChangeToLowerCase">
   <property name="text">
    <string>全小写</string>
   </property>
  </action>
  <action name="actChangeToUpperCase">
   <property name="text">
    <string>全大写</string>
   </property>
  </action>
  <action name="actMultiEditor">
   <property name="text">
    <string>Multi Editor</string>
   </property>
  </action>
  <action name="actBatchRenamer">
   <property name="text">
    <string>Batch Rename</string>
   </property>
  </action>
  <action name="actMoveListToSelection">
   <property name="text">
    <string>列表移至工程选中</string>
   </property>
  </action>
  <action name="actUndo">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="actRedo">
   <property name="text">
    <string>重做</string>
   </property>
  </action>
  <action name="actCreatePlayEvent">
   <property name="text">
    <string>创建播放事件</string>
   </property>
  </action>
  <action name="actReplaceSourceFiles">
   <property name="text">
    <string>Replace Source Files</string>
   </property>
  </action>
  <action name="actRemoveAllSwitchAssignments">
   <property name="text">
    <string>清空分配</string>
   </property>
  </action>
  <action name="actConvertToWorkUnit">
   <property name="text">
    <string>Work Unit</string>
   </property>
  </action>
  <action name="actConvertToVirtualFolder">
   <property name="text">
    <string>Virtual Folder</string>
   </property>
  </action>
  <action name="actConvertToSwitchContainer">
   <property name="text">
    <string>Switch Container</string>
   </property>
  </action>
  <action name="actConvertToRandomSequenceContainer">
   <property name="text">
    <string>Random/Sequence Container</string>
   </property>
  </action>
  <action name="actConvertToBlendContainer">
   <property name="text">
    <string>Blend Container</string>
   </property>
  </action>
  <action name="actConvertToActorMixer">
   <property name="text">
    <string>Actor Mixer</string>
   </property>
  </action>
  <action name="actBankAssignmentMatrix">
   <property name="text">
    <string>自动映射分配工具</string>
   </property>
  </action>
  <action name="actAddToSelectedBank">
   <property name="text">
    <string>添加列表到选中的Bank</string>
   </property>
  </action>
  <action name="actRenameOriginalToWwise">
   <property name="text">
    <string>重命名源文件</string>
   </property>
  </action>
  <action name="actDeleteUnusedAKDFiles">
   <property name="text">
    <string>删除多余AKD文件</string>
   </property>
  </action>
  <action name="actClearInclusions">
   <property name="text">
    <string>清除包含内容</string>
   </property>
  </action>
  <action name="actBreakContainer">
   <property name="text">
    <string>拆除结构</string>
   </property>
  </action>
  <action name="actAddToSameNameBank">
   <property name="text">
    <string>Add to Same Name Bank</string>
   </property>
  </action>
  <action name="actSetInclusionToMediaOnly">
   <property name="text">
    <string>Set Inclusion to Media Only</string>
   </property>
  </action>
  <action name="actIncludeMediaOnly">
   <property name="text">
    <string>仅媒体</string>
   </property>
  </action>
  <action name="actIncludeEventsAndStructures">
   <property name="text">
    <string>事件与结构</string>
   </property>
  </action>
  <action name="actIncludeAll">
   <property name="text">
    <string>全部</string>
   </property>
  </action>
  <action name="actApplyFaderEditsDownstream">
   <property name="text">
    <string>混音向下汇总</string>
   </property>
  </action>
  <action name="actSplitByNetRole">
   <property name="text">
    <string>按照P1/P3拆分</string>
   </property>
  </action>
  <action name="actImportFromFile">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="actExportToFile">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="actCopySelectionToList">
   <property name="text">
    <string>复制工程选中到列表</string>
   </property>
  </action>
  <action name="actWwiseSilenceAdd">
   <property name="text">
    <string>添加Wwise Silence</string>
   </property>
  </action>
  <action name="actWwiseSilenceRemove">
   <property name="text">
    <string>删除Wwise Silence</string>
   </property>
  </action>
  <action name="actReplaceEventNames">
   <property name="text">
    <string>Replace Event Names</string>
   </property>
  </action>
  <action name="actBatchReplaceTool">
   <property name="text">
    <string>模板批量替换</string>
   </property>
  </action>
  <action name="actTrimTailSilence">
   <property name="text">
    <string>裁剪末尾冗余静音</string>
   </property>
  </action>
  <action name="actFilterIncluded">
   <property name="text">
    <string>启用</string>
   </property>
  </action>
  <action name="actSetIncluded">
   <property name="text">
    <string>启用</string>
   </property>
  </action>
  <action name="actSetExcluded">
   <property name="text">
    <string>禁用</string>
   </property>
  </action>
  <action name="actFilterExcluded">
   <property name="text">
    <string>禁用</string>
   </property>
  </action>
  <action name="actTempTool">
   <property name="text">
    <string>临时工具</string>
   </property>
  </action>
  <action name="actLocalizeLanguages">
   <property name="text">
    <string>导入本地化语言</string>
   </property>
  </action>
  <action name="actReplaceParent">
   <property name="text">
    <string>替换上层</string>
   </property>
  </action>
  <action name="actUnityResourceManager">
   <property name="text">
    <string>管理UnityResource文件</string>
   </property>
  </action>
  <action name="actCreateSwitchContainer">
   <property name="text">
    <string>Switch Container</string>
   </property>
  </action>
  <action name="actCreateVirtualFolder">
   <property name="text">
    <string>Virtual Folder</string>
   </property>
  </action>
  <action name="actCreateActorMixer">
   <property name="text">
    <string>Actor Mixer</string>
   </property>
  </action>
  <action name="actCreateWorkUnit">
   <property name="text">
    <string>Work Unit</string>
   </property>
  </action>
  <action name="actCreateBlendContainer">
   <property name="text">
    <string>Blend Container</string>
   </property>
  </action>
  <action name="actCreateRandomSequenceContainer">
   <property name="text">
    <string>Random/Sequence Container</string>
   </property>
  </action>
  <action name="actSetGenericPath">
   <property name="text">
    <string>设为通用路径</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>create_bank_per_object()</slot>
  <slot>calculate_bank_size()</slot>
  <slot>connect_to_wwise()</slot>
  <slot>apply_source_edits()</slot>
  <slot>assign_switch_container()</slot>
  <slot>filter_selection()</slot>
  <slot>select_descendants_by_type()</slot>
  <slot>apply_naming_convention()</slot>
  <slot>get_selected_objects(string)</slot>
  <slot>clear_object_list()</slot>
  <slot>open_in_multi_editor()</slot>
  <slot>delete_all_objects()</slot>
  <slot>reset_source_editor()</slot>
  <slot>enable_inclusion()</slot>
  <slot>disable_inclusion()</slot>
 </slots>
</ui>
